tools/st2-self-check

```diff
@@ -18,6 +49,25 @@ else
   exit 2
 fi
 
+#Determine API HTTP Protocol
+PROTOCOL="http"
```

Comment:
Its sorta crazy you need to do all this. Anyway, lets roll with this for now - we can redesign later if needed.

Clearly something is not right with our current test design.